last_retrieved_trending.py

```
@@ -0,0 +1,30 @@
+"""Script to check last retrieval trendings in trending.csv"""
+import logging
+from datetime import datetime
+from os.path import join
+
+import pandas as pd
+
+from ytid import config as c
+from ytid.logger import setup_logging
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def main():
+    filename = join(c.DATADIR, "trending.csv")
```

Let's raise an exception if turns out the file doesn't exist